revolv/templates/project/project.html

```diff
@@ -24,7 +24,11 @@
         {% endif %}
         <ul class="button-group nav-tab-button-group">
             {% if project.is_active %}
-                <li><a class="small button success radius" href="#donate" data-reveal-id="donate-modal">Donate</a></li>
+                {% if user.is_authenticated %}
+                    <li><a id="donate-button" class="small button success radius" href="#donate" data-reveal-id="donate-modal">Donate</a></li>
+                {% else %}
+                    <li><a id="donate-button" class="small button success radius" href="/signin/?next=/project/{{project.pk}}/&reason=donate#login">Donate</a></li>
+                {% endif %}
```

Comment:
I think this could be cleaner with a variable? Something like 

```
{% if user.is_authenticated %}
{% set donate_href = '#donate' %}
{% else %}
...
{% endif %}
<li><a id="donate-button" class="small button success radius" href="{{donate_href}}" data-reveal-id="donate-modal">Donate</a></li>
```